css/style.css

```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```

Comment:
You might run into some fun issues with importing a font through a URL once `webpack` is added to the project. If you aren't married to the Poppins (he he) font, something built into the browser might save you some time and energy.

Reply:
Yeah, I was thinking I might run into an issue with that.  I can change it pretty quick if need be.